engine/src/main/java/com/blobcity/db/sql/statements/SelectExecutor.java

```diff
@@ -141,12 +138,10 @@ private String select(final String appId, CursorNode node, final String sqlStrin
             }
 
             /* Load query result from cache if present in cache */
-            if(LicenseRules.QUERY_RESULT_CACHING) {
```

Comment:
Retain this so that a further check on the cache does not have to be done if `FeatureRules.QUERY_RESULT_CACHING = false`

Reply:
Restored.